pandas/core/internals/managers.py

```diff
@@ -1965,12 +1965,18 @@ def items_overlap_with_suffix(left, lsuffix, right, rsuffix):
 
         def lrenamer(x):
             if x in to_rename:
-                return '{x}{lsuffix}'.format(x=x, lsuffix=lsuffix)
+                if lsuffix is None:
```

Comment:
Could this be simplified a bit further to something like:
```python
def lrenamer(x):
    if x in to_rename and lsuffix is not None:
        return '{x}{lsuffix}'.format(x=x, lsuffix=lsuffix)
    return x
```
Or am I overlooking something?

Reply:
nice one! thx! changed!